include/wx/msw/chkconf.h

```diff
@@ -472,6 +472,20 @@
 #   define wxUSE_POSTSCRIPT 1
 #endif
 
+#if wxUSE_GRAPHICS_CONTEXT && defined(__WXUNIVERSAL__) && !defined(wxUSE_GRAPHICS_GDIPLUS)
+#   define wxUSE_GRAPHICS_GDIPLUS 1
+#endif
+#if !wxUSE_GRAPHICS_GDIPLUS
+#   if wxUSE_GRAPHICS_CONTEXT
+#       ifdef wxABORT_ON_CONFIG_ERROR
+#           error "wxUSE_GRAPHICS_CONTEXT requires wxUSE_GRAPHICS_GDIPLUS"
+#       else
+#           undef wxUSE_GRAPHICS_GDIPLUS
+#           define wxUSE_GRAPHICS_GDIPLUS 1
+#       endif
+#   endif
+#endif
```

Comment:
This is not quite right, you could have `wxUSE_GRAPHICS_DIRECT2D=1` and `wxUSE_GRAPHICS_GDIPLUS=0`, I think.

Reply:
Okay, I will do more changes. Tim S.